src/Quizzer.API.DAL/Entities/EntityBase.cs

```diff
@@ -1,5 +1,5 @@
 ﻿namespace Quizzer.API.DAL.Entities {
-    public abstract class EntityBase {
+    public abstract record EntityBase {
```

Comment:
The main problem a wanted to say (and i guess that @BRTEK007 too) is that in CB there is misisng implementation of IEntity...
![image](https://github.com/vecerap/iw5-2023/assets/80791135/9c4f365e-b956-45d5-adfb-ea4f1fff040f)

Reply:
Ah yes I see, I fixed this issue. I decided to not let `IEntity` empty, but include Guid property inside and omit `IWithID`, because in my opinion you would mix two different things inside codebase as `IWithId` is used inside model as well I think. If this will cause future problems, I think the fix would not be too difficult to fix.